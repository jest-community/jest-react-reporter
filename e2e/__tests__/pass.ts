import * as path from 'path';
import execa from 'execa';

async function runJest() {
  return execa('jest', [], {
    all: true,
    cwd: path.resolve(__dirname, '../pass'),
    preferLocal: true,
    localDir: path.resolve(__dirname, '../../'),
  });
}

test('handles passing test', async () => {
  const { all } = await runJest();

  console.log(all);

  expect(all).toMatchSnapshot();
});
