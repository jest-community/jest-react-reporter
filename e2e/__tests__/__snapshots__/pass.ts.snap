// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`handles passing test 1`] = `
"<bold>Test Suites:</> </>0 of 1 total</>
<bold>Tests:</>       </>0 total</>
<bold>Snapshots:</>   </>0 total</>
<bold>Time:</>        </>0s, estimated 1s</>
<yellow><bold><inverse> RUNS </></></> </><bold>…</></>

<bold>Test Suites:</> </>0 of 1 total</>
<bold>Tests:</>       </>0 total</>
<bold>Snapshots:</>   </>0 total</>
<bold>Time:</>        </>0s, estimated 1s</>
<green><bold><inverse> PASS </></></> </><bold>…</></>

<bold>Test Suites:</> </><green><bold>1 passed</></>, 1 total</>
<bold>Tests:</>       </><green><bold>1 passed</></>, 1 total</>
<bold>Snapshots:</>   </>0 total</>
<bold>Time:</>        </>0s, estimated 1s</>

<bold>Test Suites:</> </><green><bold>1 passed</></>, 1 total</>
<bold>Tests:</>       </><green><bold>1 passed</></>, 1 total</>
<bold>Snapshots:</>   </>0 total</>
<bold>Time:</>        </>0s, estimated 1s</>"
`;
